packs/wechat/actions/lib/wechatutil/gettoken.py

```diff
@@ -0,0 +1,11 @@
+apiUrl = 'https://qyapi.weixin.qq.com/cgi-bin/gettoken'
+
+
+def gettoken(corpid, secret, session):
+    header = {'content-type': 'application/json; charset=utf8'}
+    payload = {"corpid": corpid, "corpsecret": secret}
+    response = session.get(apiUrl, params=payload, headers=header).json()
+    if response['access_token'] is not None:
+        return 0, response['access_token'], None
+    else:
+        return response['errcode'], None, response['errmsg']
```

Comment:
needs a newline at the end here (nb not a blank line, just a newline at end of line)